libdevcore/CommonJS.cpp

```diff
@@ -52,6 +59,18 @@ bytes unpadded(bytes _b)
 	return _b;
 }
 
+std::string unpadRight(std::string _b)
```

Comment:
This has to be used only for real string. But the input parameter is a string not an int ...
Probably should add comment to warn user to not use with a non parse int.